include/sprite.h

```diff
@@ -173,6 +172,19 @@ typedef struct {
     /* 0x28 */ Hitbox hitboxes[1];
 } Sprite /* size = 0x30 */;
 
+// TODO: Unify Sprite with variable hitbox count through a macro
```

Comment:
So this would be like `SPRITE(2)` or something?

Reply:
Yeah, something like that, but make it two:
One for definition, one for declaration.